e2eshark/onnx/operators/identity/model.py

```diff
@@ -0,0 +1,69 @@
+import numpy, torch, sys
+import onnxruntime
+from onnx import numpy_helper, TensorProto, save_model
+from onnx.helper import make_model, make_node, make_graph, make_tensor_value_info
+
+# import from e2eshark/tools to allow running in current dir, for run through
+# run.pl, commutils is symbolically linked to allow any rundir to work
+sys.path.insert(0, "../../../tools/stubs")
```

Comment:
Huh? There should be a more pythonic way to do this. Modifying the system path env var like this is weird.